docs/data/joy/customization/theme-typography/NewTypographyLevel.js

```diff
@@ -6,11 +6,10 @@ import Typography from '@mui/joy/Typography';
 const customTheme = extendTheme({
   typography: {
     kbd: {
-      background:
-        'linear-gradient(to top, var(--joy-palette-background-level2), var(--joy-palette-background-surface))',
+      backgroundColor: 'var(--joy-palette-background-surface)',
```

Comment:
When I ran `yarn docs:typescript:formatted` this file changed

Reply:
This is related to a Joy UI default theme update: https://github.com/mui/material-ui/pull/36843/files#diff-300d121fc27c24b580132ae34564f8ac77dfb0aeaeaa3dcbcce03c3ff31b7011

Can you try merging the latest master to your branch and see if this change goes away?

Reply:
Git seems to think my branch is up to date with master so merging does nothing.